libs/dh/wholesale/feature-request-calculation/src/lib/dh-wholesale-request-calculation.ts

```diff
@@ -122,10 +125,10 @@ export class DhWholesaleRequestCalculationComponent {
   private _transloco = inject(TranslocoService);
   private _toastService = inject(WattToastService);
   private _destroyRef = inject(DestroyRef);
-  private _formGroupDirective = inject(FormGroupDirective);
-  private _selectedEicFunction: EicFunction | null | undefined = null;
+  private _selectedEicFunction!: EicFunction;
```

Comment:
suggestion: the property can be `null`/`undefined` according to the return type, so let's reflect that here as well.